gatsby/content/projects/bots/matrix-eno-bot.mdx

```diff
@@ -27,9 +27,10 @@ the template and the SDK are all written in Python 3. The commands that the
 matrix-eno-bot comes with are written in bash and in python3.
 
 Intended audience/users:
-  small homeserver set up for friends and family or small community
-  tinkerers who want to learn how a bot works
-  people who want to play around with Python code and Matrix
+
+    small homeserver: set up for friends and family or small community
```

Comment:
would it make sense to use bullets here?

```
* small homeserver: set up for friends and family or small community 
```

Reply:
Yes it does. Thank you for the improvement suggestion. Cheers!

Commit fcdcac3081bb02b74191a5da5b9d1ce7b5540330 adds the bullets.